File: src/main/java/com/ormComp/model/Address.java
package com.ormComp.model;

import jakarta.persistence.*;
import lombok.Data;

import java.util.List;

@Entity
@Data
@Table(name = "addresses", schema = "rental")
public class Address {
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Id
    private long id;
    @Column(name = "city")
    private String city;
    @Column(name = "street")
    private String street;
    @Column(name = "post_code")
    private String postCode;
    @Column(name = "number")
    private String number;
    @Column(name = "country")
    private String country;
    @Column(name = "longitude")
    private double longitude;
    @Column(name = "latitude")
    private double latitude;
    @OneToMany(mappedBy = "address")
    private List<Building> building;

}

File: DotNet/Models/Address.cs
using System;
using System.Collections.Generic;

namespace Endpoints.Models;

public partial class Address
{
    public long Id { get; set; }

    public string City { get; set; } = null!;

    public string Number { get; set; } = null!;

    public string Street { get; set; } = null!;

    public string PostCode { get; set; } = null!;

    public string Country { get; set; } = null!;

    public decimal Longitude { get; set; }

    public decimal Latitude { get; set; }

    public virtual ICollection<Building> Buildings { get; set; } = new List<Building>();
}
